syntax = "proto3";

package request_models;

/**/
import "player-specialization.proto";

message ChangePlayerSpecializationRequestModel {
	int32 player_id = 1;
	util_models.PlayerSpecialization specialization = 2; 
}


/* // for godot 
import "../utils/player-specialization.proto";

message ChangePlayerSpecializationRequestModel {
	int32 player_id = 1;
	PlayerSpecialization specialization = 2; 
}*/